src/quo/components/settings/category/settings/view.cljs

```diff
@@ -7,20 +7,23 @@
     [react-native.core :as rn]))
 
 (defn settings-category
-  [{:keys [label data container-style] :as props}]
-  (let [theme         (quo.theme/use-theme)
-        settings-item (filter identity data)]
+  [{:keys [label data blur? container-style]}]
+  (let [theme          (quo.theme/use-theme)
+        settings-items (filter identity data)
```

Comment:
@ilmotta why do we need this filter here?

Reply:
This was already here and it's a way to eliminate non-truthy values from `data`. Another way to do I believe is `(remove nil? data)`, but I wanted to play safe and didn't change. I can change the code if you think the original is confusing.

Reply:
Yes, it was confusing to me, but I figured it out 😉.

<img width="251" alt="Screenshot 2024-06-20 at 16 45 37" src="https://github.com/status-im/status-mobile/assets/19279756/d2fc799c-8087-4d3a-a029-83fc9be2e575">

I think `remove` will be easier to understand. However, if you think it's risky, we can keep it.

Thank you 😊